mysql-test/suite/rocksdb/t/issue7745.test

```diff
@@ -0,0 +1,18 @@
+--source include/have_rocksdb.inc
```

Comment:
This isn't specific to rocksdb.

Could you move the testcase to the proper directory and remove this check?